docs/cookbook/prompt_engineering/chaining_based/chain_of_verification.md

```diff
@@ -68,7 +68,7 @@ async def answer(query: str): ...
     {response}
 
     Here is some fact checking on the response:
-    {verification_q_and_a}
+    {verification_q_and_a:list}
```

Comment:
this should be `:lists` and the q and a below should be a list of lists, right?

Reply:
it's `:list` because each example is formatted as: `verification_q_and_a = [f"Q:{q}\nA:{a}" for q, a in zip(verification_questions, verification_answers)]`

Reply:
fixed with below